tests/python/gaia-ui-tests/gaiatest/apps/settings/regions/addons.py

```diff
@@ -44,3 +54,16 @@ def tap_first_item(self):
 
     def toggle_addon_status(self):
         self.marionette.find_element(*self._state_toggle_locator).tap()
+
+
+class Addon(PageRegion):
+
+    _name_locator = (By.CSS_SELECTOR, 'span')
+
+    @property
+    def addon_name(self):
```

Comment:
Nit: `name` is enough, the class is already called `Addon`, then the common usage would be `addon.name`, instead of `addon.addon_name`